agent/php_execute.c

```diff
@@ -1685,6 +1685,68 @@ void nr_php_user_instrumentation_from_opcache(TSRMLS_D) {
  */
 #if ZEND_MODULE_API_NO >= ZEND_8_0_X_API_NO /* PHP8+ */
 
+static void nr_php_observer_attempt_call_cufa_handler(NR_EXECUTE_PROTO) {
+  NR_UNUSED_FUNC_RETURN_VALUE;
+  if (NULL == execute_data->prev_execute_data) {
```

Comment:
is `execute_data` itself always guaranteed to be non-null here?

Reply:
Can I get @zsistla 's input here?

Reply:
There is a check in the oapi branch further up the call stack. So unless it will be modified in between it should be fine.